src/utils/encoding.js

```diff
@@ -32,6 +32,9 @@ export const base64ToHex = (bytes: string) => asciiToHex(base64.decode(bytes));
 
 export const hexToBase64 = (hex: string) => base64.encode(hexToAscii(hex));
 
+export const objToBase64 = (obj: Object | Array<*>): string =>
+  base64.encode(unescape(encodeURIComponent(JSON.stringify(obj))));
```

Comment:
This is a pretty funny-looking sequence (encode, then *unescape*, then different encode?). It should have a comment pointing to an explanation. :-)

This one is pretty good:
https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding

Also, I'd rather separate this into two pieces:
* JSON.stringify -- the caller can do this, it knows its business
* this funny dance to turn what's already a perfectly good string, into something that will be inert under percent-decoding and basically any other reinterpretation.

I'd also want the name to be a bit more specific, because there's more going on to what this encoding means than "convert to base64". It's converting to base64 in a very particular way, which is that it's turning the string into UTF-8 (through this hack of `unescape(encodeURIComponent(...))`), and base64-encoding that bytestream. If the code interpreting this only knew "it's base64", that wouldn't be enough to interpret it correctly with confidence.

So I think the primitive I'd want here is something like this:

`export const base64Utf8Encode  = (text: string): string => base64.encode(unescape(encodeURIComponent(text)))`